.mci.yml

```diff
@@ -175,6 +175,9 @@ functions:
                     git clone git@github.com:mongodb-labs/drivers-evergreen-tools.git
                 fi
                 cd drivers-evergreen-tools
+                if [[ "${build_variant}" =~ "rhel9" ]]; then
+                    export SKIP_LEGACY_SHELL=1
```

Comment:
Please add a short comment describing the RHEL 9 specific issue that this is addressing (e.g. "# Avoid blah error").